auditbeat/docs/modules/auditd.asciidoc

```diff
@@ -225,6 +229,11 @@ time.
 - `none`: No backpressure mitigation measures are enabled.
 --
 
+*`reload`*:: `enabled` auditbeat check auditd rule files in fixed interval 
+`period`, reload auditd rule on host if any changes to the rule file detected.
+Its similar to the config reload, but instead of .yml files it is for .rules
+files.
```

Comment:
Same as above, these changes should go in: `auditbeat/module/auditd/_meta/docs.asciidoc`. In addition, after both this and the above change are made, you'll need to run `mage update` in the auditbeat directory